cmd/create/cluster/cmd.go

```diff
@@ -3896,6 +3932,11 @@ func buildCommand(spec ocm.Spec, operatorRolesPrefix string,
 		command += " --no-cni"
 	}
 
+	if len(spec.AdditionalAllowedPrincipals) > 0 {
+		command += fmt.Sprintf(" --additional-allowed-principals %s",
+			strings.Join(spec.AdditionalAllowedPrincipals, ","))
+	}
+
```

Comment:
Got a question for this one. Do we need `--additional-allowed-principals principal1, principal 2` or ``--additional-allowed-principals principal1, --additional-allowed-principals principal 2`